src/portable/st/stm32_fsdev/dcd_stm32_fsdev.c

```diff
@@ -323,29 +323,27 @@ void dcd_remote_wakeup(uint8_t rhport)
   remoteWakeCountdown = 4u; // required to be 1 to 15 ms, ESOF should trigger every 1ms.
 }
 
-// I'm getting a weird warning about missing braces here that I don't
```

Comment:
@pigrew wMaxPacketSize is a struct, hence the warning :)

Reply:
Thanks for noticing that...